docs/styles/position.stories.mdx

```diff
@@ -0,0 +1,91 @@
+import { Meta, Story } from "@storybook/addon-docs/blocks";
+import { TableLayout } from "components/util/Table"
+import FDS from 'lib/dictionary/js/styleConstants';
+import { Item, generalStyles, containerStyles } from './styledocUtils';
+
+export const positionStyles = {
+  ...generalStyles,
+  border: `1px solid ${FDS.COLOR_CHARCOAL}`,
+  padding: 2 
+}
+
+export const boxStyles = {
+  height: 30,
+  width: 30,
+  background: FDS.COLOR_CHARCOAL,
+  borderRadius: 3
+}
+
+<Meta title="Style Utilities/Position"  />
+
+## Child Align
+Positions direct child(s) of element using flex properties.
+**These classes modify `flex` properties.**
+Avoid combining `alignChild` classes with flexbox layout elements.
+
+Signature: `alignChild--(x-axis)--(y-axis)`
+
+<Story name="Child Align">
+  <>
+    <TableLayout
+      isCSS
+      headers={['class', 'description']}
+      rows={[
+        ['alignChild--left--top', 'Adds standard margin to top of element'],
+        ['alignChild--left--center', 'Adds standard margin to bottom of element'],
+        ['alignChild--left--bottom', 'Adds standard margin to right of of element'],
+        ['alignChild--right--top', 'Adds standard margin to left of element'],
+        ['alignChild--right--center', 'Adds standard margin to all sides'],
+        ['alignChild--right--bottom', 'Adds standard margin to all sides'],
+        ['alignChild--center--top', 'Adds standard margin to all sides'],
+        ['alignChild--center--center', 'Adds standard margin to all sides'],
+        ['alignChild--center--bottom', 'Adds standard margin to all sides'],
+      ]} 
+    />
+    <div style={containerStyles}>
+      <Item label="center--center"><div className="alignChild--center--center" style={positionStyles}><div style={boxStyles}/></div></Item>
+      <Item label="center--top"><div className="alignChild--center--top" style={positionStyles}><div style={boxStyles}/></div></Item>
+      <Item label="center--bottom"><div className="alignChild--center--bottom" style={positionStyles}><div style={boxStyles}/></div></Item>
+      <Item label="left--center"><div className="alignChild--left--center" style={positionStyles}><div style={boxStyles}/></div></Item>
+      <Item label="left--top"><div className="alignChild--left--top" style={positionStyles}><div style={boxStyles}/></div></Item>
+      <Item label="left--bottom"><div className="alignChild--left--bottom" style={positionStyles}><div style={boxStyles}/></div></Item>
+      <Item label="right--center"><div className="alignChild--right--center" style={positionStyles}><div style={boxStyles}/></div></Item>
```

Comment:
I can see arguments both for and against what I'm about to suggest...

Each one of these examples repeats most of the same code. Do we want to stick with the declarative, repetitive pattern, or have each MDX file declare some custom example wrapper?

For example, in this MDX file we could do...

```
const AlignChildExample = ({ x, y }) =>
	<Item label={`${x}--${y}`}><div className={`alignChild--${x}--${y}`} style={{...generalStyles, etc}}><div style={{height: 30, etc}} /></div></Item>;
```

Then in the example, we can just do
```
<AlignChildExample x="left" y="top" />
<AlignChildExample x="left" y="center" />
etc
```

Reply:
Some of this is answered by this comment: https://github.com/cbinsights/form-design-system/pull/276#discussion_r390375109

Essentially to summarize:

* This pattern I'm probably only using for like 2 stories, so I feel like the abstraction isn't great, which is why you see me using styles a lot.
* If some of the examples move to inside the table (which I feel I could make a great argument for, and I could complete some of that when I get fatigued from whatever bigger project I'm working), it nullifies both these styles and components. I figure better to put it out there, figure out what users like, get this initiative done, and then slowly layer onto it later. 

But yeah I sort of don't see the benefit in creating an abstraction around this now. To do that, I'd probably would need to make a component that supports the 6 or so different variations of examples I'm using, that again may go away at some point.

Reply:
To be clear, I'm not advocating for a general use example component, just defining a context-specific component when helpful (in the same file it's used).

This is optional though, I don't want to hold this up with over-optimizing things

Reply:
oic. Yeah I could do that. I guess the only question there, is if we'd want to make a custom component like that for other places. The value of accepting the direction as a prop IMO is only useful given that the label is still there, and I'm sort of thinking these examples "may" fit better in the table. I'll punt on it for now if it's ok with you, but definitely want to revisit this later on.